app/models/interval_receiver.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 
 class IntervalReceiver < EventReceiver
-  def dispatch!
-  end
-
   def trigger_condition_met?(job)
```

Comment:
What do we want to do for the `InternalReceiver` @davepagurek?

Reply:
just `true`, since it will only be called when its corresponding sidekiq job gets called